src/cypress-runner.ts

```diff
@@ -183,7 +183,12 @@ function getCypressOpts(
     opts.key = runCfg.cypress.key;
   }
 
-  opts = configureReporters(runCfg, opts);
+  if (runCfg.cypress.reporters) {
+    opts = configureReporters(runCfg, opts);
+    console.log(
+      'Configuring multi reporters with saucectl is deprecated. We recommend migrating your configuration to your cypress config file.',
```

Comment:
Just a bit more assertive, because we _will_ remove it and it _will_ break for them if they don't:
```suggestion
      'Configuring multi reporters with saucectl is deprecated and will be removed in a future release. Migrate your configuration to your cypress config file.',
```